modules/nf-core/samtools/view/meta.yml

```
@@ -43,6 +43,10 @@ input:
         type: file
         description: Optional file with read names to output only select alignments
         pattern: "*.{txt,list}"
+  - - index_format:
+        type: string
+        description: Index format, used together with ext.args = '--write-index'
+        pattern: "*.{bai,csi,crai}"
```

This is a val input, the pattern should be presumably 
 
```suggestion
        pattern: "bai|csi|crai"
```

?